extensions/typescript-language-features/package.nls.json

```diff
@@ -76,10 +76,10 @@
 	"typescript.preferences.importModuleSpecifierEnding.minimal": "Shorten `./component/index.js` to `./component`.",
 	"typescript.preferences.importModuleSpecifierEnding.index": "Shorten `./component/index.js` to `./component/index`.",
 	"typescript.preferences.importModuleSpecifierEnding.js": "Do not shorten path endings; include the `.js` extension.",
-	"typescript.preferences.includePackageJsonAutoImports": "Enable/disable processing `package.json` dependencies for available auto imports.",
-	"typescript.preferences.includePackageJsonAutoImports.all": "Include all listed dependencies.",
-	"typescript.preferences.includePackageJsonAutoImports.excludeDev": "Exclude devDependencies.",
-	"typescript.preferences.includePackageJsonAutoImports.none": "Disable package.json dependency processing.",
+	"typescript.preferences.includePackageJsonAutoImports": "Enable/disable searching `package.json` dependencies for available auto imports.",
+	"typescript.preferences.includePackageJsonAutoImports.auto": "Search dependencies based on estimated performance impact.",
+	"typescript.preferences.includePackageJsonAutoImports.on": "Always search dependencies.",
```

Comment:
Just a quick question, will this also search dev dependencies?

Reply:
No, I decided to remove that functionality unless/until there’s high demand for it. Since lots of devDependencies go unimported and creating auto import providers that contain lots of dependencies have an associated startup performance penalty, the setting was a perf trap door without enough room to explain all the considerations of enabling it.